src/main/java/reo/Storage.java

```diff
@@ -0,0 +1,177 @@
+package reo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/** Supports the required file operations for reo.Reo. */
+public class Storage {
+    private String filePath;
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+    /**
+     * Returns true if the parameter is 1, and false otherwise.
+     *
+     * @param i The integer used to represent true or false in the saved file.
+     * @return Boolean representation of i.
+     */
+    public static boolean convertIntToBool(int i) {
+        return i == 1;
+    }
```

Comment:
Instead of doing this conversion, you can do an if check after the switch statement to check whether split[1] is equal to 1 and mark the task accordingly.